src-tauri/src/main.rs

```diff
@@ -329,19 +329,21 @@ fn trigger_action(
                         whole_buffer.push(response.clone());
                         delta_buffer.push(response);
 
-                        let delta_output = {
-                            if delta_buffer.len() > 4 {
```

Comment:
I regret not writing comments so much... I have no idea why I did this, and had `rem` disabled b/c I was hacking on it 🤦 . I will try to go spelunk in history...

Reply:
I can undo the enter thing... got it locally. Want me to push? Definitely skipping new lines for me tho...

list the meanings of life

results in

1. Meaning in a philosophical sense: The purpose or significance of human existence, often explored through existentialism and other philosophies.2. Scientific meaning: Biological functions such as reproduction and survival.3. Personal meaning: Subjective understanding and personal beliefs about one's role and place in the world.4. Spiritual meaning: Interpretations derived from religious or spiritual beliefs, often related to connection with a higher power, enlightenment, or spiritual growth.5. Collective meaning: Societal values, norms, and cultural traditions that shape our understanding of life's purpose.

Reply:
gotta say all the things I'd actually want to use this for are text areas and not fields and are terribly crippled by no new line.

Reply:
Easy to fix by adding a user setting (allow enabling "simulate newlines with Enter key").

Strange that you don't have newlines though - they seem to work for me.

What system are you on?

Reply:
(for example https://www.loom.com/share/d3adddc37c2e4c8daeef11255c0135ea?sid=5ec9ee09-5b9d-4222-aeff-f82cf241db74)

Reply:
mac